docs/random-numbers.md

```diff
@@ -107,9 +124,11 @@ This is not a problem in NumPy, which always evaluates code in the order defined
 In JAX, however, this is more problematic: for efficient execution, we want the JIT compiler to be free to reorder, elide, and fuse various operations in the function we define.
 Further, when executing in multi-device environments, execution efficiency would be hampered by the need for each process to synchronize a global state.
 
+The Mersenne Twister PRNG is also known to have a [number](https://cs.stackexchange.com/a/53475) of problems, it has a large 2.5kB state size, which leads to problematic [initialization issues](https://dl.acm.org/citation.cfm?id=1276928).  It [fails](http://www.pcg-random.org/pdf/toms-oneill-pcg-family-v1.02.pdf) modern BigCrush tests, and is generally slow.
```

Comment:
I think we can't put this paragraph here: the next section starts with "to avoid these issues", but "these issues" refer to the previous paragraph, which has now been bumped out of the reader's L2 cache.

Maybe we can just remove this?

Reply:
Done. (At first I had thought "these issues" could also refer to the problems with the Mersenne Twister PRNG, but now I see how that doesn't apply here).